bin/pg_repack.c

```diff
@@ -2222,21 +2219,22 @@ pgut_help(bool details)
 		return;
 
 	printf("Options:\n");
-	printf("  -a, --all                 repack all databases\n");
-	printf("  -t, --table=TABLE         repack specific table only\n");
-	printf("  -I, --parent-table=TABLE  repack specific parent table and its inheritors\n");
-	printf("  -c, --schema=SCHEMA       repack tables in specific schema only\n");
-	printf("  -s, --tablespace=TBLSPC   move repacked tables to a new tablespace\n");
-	printf("  -S, --moveidx             move repacked indexes to TBLSPC too\n");
-	printf("  -o, --order-by=COLUMNS    order by columns instead of cluster keys\n");
-	printf("  -n, --no-order            do vacuum full instead of cluster\n");
-	printf("  -N, --dry-run             print what would have been repacked\n");
-	printf("  -j, --jobs=NUM            Use this many parallel jobs for each table\n");
-	printf("  -i, --index=INDEX         move only the specified index\n");
-	printf("  -x, --only-indexes        move only indexes of the specified table\n");
-	printf("  -T, --wait-timeout=SECS   timeout to cancel other backends on conflict\n");
-	printf("  -D, --no-kill-backend     don't kill other backends when timed out\n");
-	printf("  -Z, --no-analyze          don't analyze at end\n");
-	printf("  -k, --no-superuser-check  skip superuser checks in client\n");
-	printf("  -C, --exclude-extension   don't repack tables which belong to specific extension\n");
+	printf("  -a, --all                   repack all databases\n");
+	printf("  -t, --table=TABLE           repack specific table only\n");
+	printf("  -I, --parent-table=TABLE    repack specific parent table and its inheritors\n");
+	printf("  -c, --schema=SCHEMA         repack tables in specific schema only\n");
+	printf("  -s, --tablespace=TBLSPC     move repacked tables to a new tablespace\n");
+	printf("  -S, --moveidx               move repacked indexes to TBLSPC too\n");
+	printf("  -o, --order-by=COLUMNS      order by columns instead of cluster keys\n");
+	printf("  -n, --no-order              do vacuum full instead of cluster\n");
+	printf("  -N, --dry-run               print what would have been repacked\n");
+	printf("  -j, --jobs=NUM              Use this many parallel jobs for each table\n");
+	printf("  -i, --index=INDEX           move only the specified index\n");
+	printf("  -x, --only-indexes          move only indexes of the specified table\n");
+	printf("  -T, --wait-timeout=SECS     timeout to cancel other backends on conflict\n");
+	printf("  -D, --no-kill-backend       don't kill other backends when timed out\n");
+	printf("  -Z, --no-analyze            don't analyze at end\n");
+	printf("  -k, --no-superuser-check    skip superuser checks in client\n");
+	printf("  -C, --exclude-extension     don't repack tables which belong to specific extension\n");
+	printf("  -m, --max-swap-tuples=NUM   max tuples allowed in replay log at table swap\n");
```

Comment:
`trigger table swap once the last batch replayed by this was of size NUM or less` might be more accurate, but harder to understand.

Mentioning `(default=20)` might help end users understand what changing this would do